assets/report.json

```diff
@@ -5336,12 +5314,10 @@
     "": {
       "summary": {
         "max_points": 16,
-        "impl_points": 15,
+        "impl_points": 16,
         "stub_penalty": 0
       },
-      "missing": [
-        "isXMLName()"
```

Comment:
This is stubbed right now- from the last PR, it seems that your tool doesn't mark methods that aren't defined on classes (e.g. `flash.crypto.getRandomBytes`, `flash.net.registerClassAlias`) as stubs.

Reply:
Fixed